src/todolist.js

```diff
@@ -0,0 +1,32 @@
+export default class ThingsTodo {
+  constructor() {
+    if (localStorage.getItem('todoList')) {
+      this.list = JSON.parse(localStorage.getItem('todoList'));
+    } else {
+      this.list = [];
+    }
+  }
+
+  addList(lsList) {
+    this.list.push(lsList);
+    localStorage.setItem('todoList', JSON.stringify(this.list));
+  }
+
+  deleteList(deleteID) {
+    this.list = this.list.filter((lsList) => lsList.index !== deleteID);
+    this.list.forEach((lsList, index) => {
+      lsList.index = index + 1;
+    });
+    localStorage.setItem('todoList', JSON.stringify(this.list));
+  }
+
+  editList(editId, newDescription) {
+    const newTodoDesc = this.list.map((todoArray) => {
+      if (todoArray.index === editId) {
+        return { ...todoArray, description: newDescription };
+      }
+      return todoArray;
+    });
+    localStorage.setItem('todoList', JSON.stringify(newTodoDesc));
+  }
+}
```

Comment:
- [ ] Kindly make sure once you edit a task it also update data in the local storage as required for the project. For now, in your project, you can edit a task but if you reload the page it get the previous data as in the local storage data are not updated once you did it from the form.

Reply:
This change is still yet to be implemented.

Reply:
hello peter could you please explain more